### ISSUE TITLE
Text in TextVisual example does not stay centred

### ISSUE THREAD

**Issue by QuLogic** [vispy/vispy#975]
The `TextVisual` anchors to the centre by default, plus the resize handler moves the text position to be in the centre of the screen. However, as [you can see](http://youtu.be/FqJv5L8RPm4), the text does _not_ stay centred. Printing out `self.size` seems to show that the window size is determined correctly, but the text does not seem to be in the right place.

**Comment by Eric89GXL** [comment]
I suspect this will be fixed by #928 because it changes a lot about the transform system

**Comment by QuLogic** [comment]
Not just yet...
```
$ python3 text_visual.py 
Traceback (most recent call last):
  File "text_visual.py", line 41, in <module>
    c = Canvas()
  File "text_visual.py", line 15, in __init__
    self.text = visuals.TextVisual('', bold=True)
  File "/home/elliott/code/vispy/vispy/visuals/text/text.py", line 374, in __init__
    Visual.__init__(self, **kwargs)
  File "/home/elliott/code/vispy/vispy/visuals/visual.py", line 165, in __init__
    self._prepare_transforms(self)
  File "/home/elliott/code/vispy/vispy/visuals/visual.py", line 234, in _prepare_transforms
    raise NotImplementedError()
NotImplementedError
```
